x-pack/plugins/apm/public/components/shared/service_icons/index.tsx

```diff
@@ -30,6 +32,22 @@ const cloudIcons: Record<string, string> = {
   azure: 'logoAzure',
 };
 
+function getServerlessTitle(serverlessType?: ServerlessType): string {
+  if (serverlessType === ServerlessType.AWS_LAMBDA) {
+    return i18n.translate('xpack.apm.serviceIcons.aws_lambda', {
+      defaultMessage: 'AWS Lambda',
+    });
+  } else if (serverlessType === ServerlessType.AZURE_FUNCTIONS) {
+    return i18n.translate('xpack.apm.serviceIcons.azure_functions', {
+      defaultMessage: 'Azure Functions',
+    });
+  } else {
+    return i18n.translate('xpack.apm.serviceIcons.serverless', {
+      defaultMessage: 'Serverless',
+    });
+  }
+}
```

Comment:
I think a `switch` would be a bit clearer, WDYT?

```suggestion
 switch(serverlessType){
   case ServerlessType.AWS_LAMBDA: {
     return i18n.translate('xpack.apm.serviceIcons.aws_lambda', {
      defaultMessage: 'AWS Lambda',
    });
   }
   case ServerlessType.AZURE_FUNCTIONS: {
     return i18n.translate('xpack.apm.serviceIcons.azure_functions', {
      defaultMessage: 'Azure Functions',
    });
   }
   default:{
    return i18n.translate('xpack.apm.serviceIcons.serverless', {
      defaultMessage: 'Serverless',
    });
   }
 }
}
```